database/migrations/20200208123841_SKILLED_IMPACT_REQUESTS.js

```diff
@@ -0,0 +1,68 @@
+const skills = [
```

Comment:
Since @dacruz21's PR (#100) already created an enum, we can delete this array and reference that enum in this migration (looks like you're already using the same enum name but double-check).